src/App.tsx

```diff
@@ -23,6 +31,29 @@ const App = () => {
     setUserUID(userUID);
   };
 
+  const fetchCurrentUser = async () => {
+    try {
+      const userRef = doc(db, "test-tribe", "DrJZcEmb22Z5pG6fn2Fj2YYTHEy1");
+      const userDocSnap = await getDoc(userRef);
+      const user = userDocSnap.data() as DocumentData;
```

Comment:
You should be able to cast this `const user = userDocSnap.data() as UserProfile;` which means you do not need to create the object below it.